KippysEscape.uproject

```diff
@@ -34,6 +33,10 @@
 				"HoloLens",
 				"Win64"
 			]
+		},
+		{
+			"Name": "EditorScriptingUtilities",
```

Comment:
Do you know what brought in this new dependency?

Reply:
Some scripts. EditorScriptingUtilities code initially were in the engine, but was moved to a plugin for 4.26.